rest_framework_simplejwt/settings.py

```diff
@@ -14,6 +14,7 @@
     'REFRESH_TOKEN_LIFETIME': timedelta(days=1),
     'ROTATE_REFRESH_TOKENS': False,
     'BLACKLIST_AFTER_ROTATION': True,
+    'JTI_CLAIM_IS_MENDATORY': True,
```

Comment:
Mandatory, not mendatory

Reply:
It was necessary for my application as we have to authenticate users with a JWT token without a JTI claim. We don't own the JWT creation which is done by another service and JWT comes without JTI claim.
We use django-rest-framework-simplejwt only to verify token with the private key (this is the important part of the authentication for me, not the jit) and receive an authenticated user. 
We don't use the token creation part.